virttest/utils_libvirt/libvirt_nested.py

```diff
@@ -0,0 +1,80 @@
+import logging
+
+from avocado.core import exceptions
+from avocado.utils import process
+
+from virttest import cpu
+from virttest import utils_libvirtd
+from virttest import utils_package
+
+
+def install_virt_pkgs(vm_session):
+    """
+    Install required virt packages for nested virt test
+
+    :param vm_session: session to the vm
+    :raises: exceptions.TestError if installation fails
+    """
+    pkg_names = ['libvirt', 'qemu-kvm']
+    logging.debug("Virt packages will be installed")
+    pkg_mgr = utils_package.package_manager(vm_session, pkg_names)
+    if not pkg_mgr.install():
+        raise exceptions.TestError("Package '%s' installation "
+                                   "fails" % pkg_names)
+    libvirtd = utils_libvirtd.Libvirtd()
```

Comment:
I don't quite understand why you restart libvirtd on host, would you mind adding some explanations?  libvirt* packages are installed on VM, so do you need to restart libvirtd service on the VM?

Reply:
Yes, you are right. That's not my original purpose. I will update it.

Reply:
updated @Yingshun